ballerina/build.gradle

```diff
@@ -785,7 +798,11 @@ task testExamples() {
             'task-service-appointment',
             'error-handling-in-single-place',
             'file-io',
-            'uuid'  // Output varies
+            'uuid',  // Output varies
+
+            // Disabling temporarily (https://github.com/ballerina-platform/ballerina-distribution/issues/1242)
+            "configurable",
```

Comment:
```suggestion
```

I believe this should be fixed with @HindujaB's changes.

Reply:
yes.  #1173